sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1337,11 +1337,10 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
 
   def duplicateMapKeyFoundError(key: Any): SparkRuntimeException = {
     new SparkRuntimeException(
-      errorClass = "_LEGACY_ERROR_TEMP_2127",
+      errorClass = "DUPLICATED_MAP_KEY",
       messageParameters = Map(
         "key" -> key.toString(),
-        "mapKeyDedupPolicy" -> toSQLConf(SQLConf.MAP_KEY_DEDUP_POLICY.key),
-        "lastWin" -> toSQLConf(SQLConf.MapKeyDedupPolicy.LAST_WIN.toString())))
+        "mapKeyDedupPolicy" -> toSQLConfVal(SQLConf.MAP_KEY_DEDUP_POLICY.key)))
```

Comment:
This is not a value, it is a config name, correct?
```suggestion
        "mapKeyDedupPolicy" -> toSQLConf(SQLConf.MAP_KEY_DEDUP_POLICY.key)))
```

Reply:
Ah, now I got the difference.
Thanks for the correction!